updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc

```diff
@@ -0,0 +1,136 @@
+
+[id="update-service-prereqs"]
+== Prerequisites
+
+* Have access to the internet to obtain the necessary container images.
+* Have write access to a container registry in the restricted-network environment to push and pull images. The container registry must be compatible with Docker registry API v2.
+* You must have the `oc` command-line interface (CLI) tool installed.
+* For more information on installing Operators, see xref:../../operators/user/olm-installing-operators-in-namespace.adoc#olm-installing-operators-in-namespace[Installing Operators in your namespace].
+
+[id="registry-configuration-for-update-service"]
+== Configuring access to a secured registry for the OpenShift update service
+
+If the release images are contained in a secure registry, complete the steps in xref:../../registry/configuring-registry-operator.adoc#images-configuration-cas_configuring-registry-operator[Configuring additional trust stores for image registry access] along with following changes for the update service.
+
+The OpenShift Update Service Operator needs the config map key name `updateservice-registry` in the registry CA cert.
+
+.Image registry CA config map example for the update service
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: my-registry-ca
+data:
+  updateservice-registry: | <1>
+    -----BEGIN CERTIFICATE-----
+    ...
+    -----END CERTIFICATE-----
+  registry-with-port.example.com..5000: | <2>
+    -----BEGIN CERTIFICATE-----
+    ...
+    -----END CERTIFICATE-----
+----
+<1> The OpenShift Update Service Operator requires the config map key name updateservice-registry in the registry CA cert.
+<2>  If the registry has the port, such as `registry-with-port.example.com:5000`, `:` should be replaced with `..`.
+
+include::modules/images-update-global-pull-secret.adoc[leveloffset=+1]
+
+[id="update-service-install"]
+== Installing the OpenShift Update Service Operator
+
+To install the OpenShift Update Service, you must first install the OpenShift Update Service Operator by using the {product-title} web console or CLI.
+
+[NOTE]
+====
+For clusters that are installed on restricted networks, also known as disconnected clusters, Operator Lifecycle Manager by default cannot access the Red Hat-provided OperatorHub sources hosted on remote registries because those remote sources require full internet connectivity. For more information, see xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks].
+====
+
+include::modules/update-service-install-web-console.adoc[leveloffset=+2]
+
+include::modules/update-service-install-cli.adoc[leveloffset=+2]
+
+include::modules/update-service-graph-data.adoc[leveloffset=+1]
+
+[id="update-service-mirror-release_updating-restricted-network-cluster"]
+== Mirroring the {product-title} image repository
```

Comment:
It's not clear to me why we append `repository` in the section name. Other than images, do we mirror the image repo?

Reply:
I'm not sure, this was how the section was titled when I moved it over. @LalatenduMohanty or @wking, do you know any context about this?